meka/srcs/sound/sound.h

```diff
@@ -8,7 +8,7 @@
 //-----------------------------------------------------------------------------
 
 #define SOUND_BUFFERS_COUNT         (4)
-#define SOUND_BUFFERS_SAMPLE_COUNT  (1024)
+#define SOUND_BUFFERS_SAMPLE_COUNT  (4096)

```

Comment:
Does this increase audio latency?

Reply:
Good catch. It does.
And it's actually quite noticeable, sorry for missing it. I was just listening to BGMs to test this.

The only reason for this increase is that, at extremely low output framerates (20Hz and 10Hz), the number of samples being requested occasionally exceeds the total buffer capacity.
And actually, since the sound is so wonky at lower rates you don't exactly notice the skip.

I will try testing this a little more without the buffer increase to try and figure out how bad the issue is at lower framerates, and push an update without the increase if things seem relatively OK.